cpu/cpu_windows.go

```diff
@@ -1,6 +1,8 @@
 package cpu
 
 import (
+	"strconv"
+
 	utils "github.com/DataDog/gohai/windowsutils"
 	// "strconv"
```

Comment:
nitpick: could you remove this comment and move the `strconv` import after `regexp` (alphabetical order)?